recipes/mbits-args/all/conanfile.py

```diff
@@ -36,8 +36,8 @@ class MBitsArgsConan(ConanFile):
         "argument-parsing",
     )
     settings = "os", "compiler", "build_type", "arch"
-    options = {"fPIC": [True, False]}
-    default_options = {"fPIC": True}
+    options = {"fPIC": [True, False], "shared": [True, False, "deprecated"]}
```

Comment:
Seems odd to add an option that does nothing 🤔

Reply:
The idea is that this recipe used to have a shared option, but the last PR by its author removed it without first waiting for a "deprecation" period (Which at first we thought was a typo in the original PR)
If we're ok with it having fell thru the cracks we can always close this PR :)

Reply:
I think it fine, we need to be more careful in the reviews though